File: Php/Warrior.php
<?php 
require_once ("loadingClass.php");

class Warrior extends Caracter{

    public function __construct ($pseudo, $age) {
        parent::__construct($pseudo, $age);
        $this -> setRace ("Warrior");
    }

    public function fight () {
        if ($this -> weapons = "Sword") {
            echo "</br> I can use this Sword";
        }
    }

}
?>

File: Java POO/Warrior.java
package game;

class Warrior extends Person {
	
	public Warrior (String pseudo, int age) {
		super (pseudo,  age);
		this.setRace("Warrior"); 
	}
	
	public void fight () {
		if (this.weapons.equals("Sword")) {
			System.out.println("I can use that");
		} 
	}
	
	
	
}